packages/dapp/package.json

```diff
@@ -12,7 +12,7 @@
     "start": "next dev",
     "clean": "rimraf next types node_modules",
     "build:next": "next build",
-    "build:types": "typechain --show-stack-traces --target ethers-v5 ../contracts/out/**/!(test*|Test*|*.t.sol|*.s.sol)/*.json --out-dir=types"
+    "build:types": "typechain --show-stack-traces --target ethers-v5 '../contracts/out/**/!(test*|Test*||*.t.sol|*.s.sol|access*)/**/!(*IAccessControl).json' --out-dir types"
```

Comment:
why do we have to pass a interface name to a build command, this one seems "hacky"

Reply:
This is very hacky lmao, basically building types is failing due to having the IAccessControl interface locally and having one inside openzeppelin and I couldn't think of another way to do it without removing the local interface

Reply:
Open to suggestions on this if you have any